### DISCUSSION TITLE
Postgres Migrations RLS?

### DISCUSSION THREAD

**Discussion by marcustut** [prisma/prisma-client-go#466] [Q&A] [answered]
I saw migrstions file are written in .prisma format. I just want to know what if I wanted to enable RLS (Row Level Security) or I want to write a trigger/policy sepcific to a table. How would I do that in `.prisma` files?

Sorry, I'm quite new to databases, I'm not sure will database migrations should actually include trigger/policy or they are written separately.

**Comment by steebchen** [comment, marked as answer]
Just the declarative schema is written in .prisma files. When you apply migrations to a database, they are run via SQL or saved in .sql files. If a feature is not currently supported in Prisma yet, you can migrate your database yourself using SQL, and then use `prisma introspect` ([docs](https://www.prisma.io/docs/concepts/components/introspection)) to pull in the database schema to the .prisma file.

**Reply by marcustut** [reply]
Thanks for the quick reply, this does clear things up for me. Would really hope Prisma's Go Client get into stable release soon, was thinking to use this tool for an upcoming project. Anyways, thanks for the great work!

**Reply by steebchen** [reply]
We don't know when it will be stable yet, but I suggest to try out the Go client anyway. It lacks behind in features but the core features work, and it uses the same query engine behind the scenes which is used by the JS/TS client.

**Reply by marcustut** [reply]
Sure, I'll give it a go on one of my personal project
